applications/vessel-history/src/features/i18n/i18n.ts

```diff
@@ -3,13 +3,21 @@ import Backend from 'i18next-http-backend'
 import LanguageDetector from 'i18next-browser-languagedetector'
 import { initReactI18next } from 'react-i18next'
 import { Locale } from 'types'
+import { WORKSPACE_ENV } from 'data/config'
 
 export const LocaleLabels = [
   { id: Locale.en, label: 'English' },
   // { id: Locale.es, label: 'Español' },
   { id: Locale.fr, label: 'Français' },
 ]
 
+const PACKAGE_NAMESPACES = ['flags']
+const GITHUB_LABELS_BRANCH = WORKSPACE_ENV === 'development' ? 'develop' : 'master'
+export const SHARED_LABELS_PATH =
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost:8000'
+    : `https://raw.githubusercontent.com/GlobalFishingWatch/frontend/${GITHUB_LABELS_BRANCH}/packages/i18n-labels`
+
```

Comment:
Is this going to work on local dev environment  cloud dev environment and cloud prod?

Reply:
I think so because in `dev` the `NODE_ENV` will be production and the `GITHUB_LABELS_BRANCH` should be develop due `WORKSPACE_ENV` let's merge so we can find it out !